### PR TITLE
updating readme.md: updating the ingestion mechanism and adding fields mapping of v27.12

### PR DESCRIPTION
**Pull request by poriya-crestdata**
Updating the ingestion mechanism and adding fields mapping of v27.12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,27 +4,110 @@ This document describes how you can collect Corelight Sensor logs by configuring
...
+
+![Deployment architecture](images/parser-corelight-architecture-without-forwarder.png)
 
 The architecture diagram shows the following components:
 
-* **Corelight Sensor**: The system running the [Corelight Sensor](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
+*  **Corelight Sensor**: The system running the [Corelight Sensor ](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
 
-* **The Corelight Sensor exporter**: The [Corelight Sensor exporter](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html) collects log data from the Sensor, and forwards it to the Chronicle forwarder.
+*  **Corelight Sensor exporters**: The [Corelight Sensor exporter](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html) collects log data from the Sensor, and forwards it to the Google Security Operations.
```
Comment (jlagermann) [inline README.md:29]: Please change this line to:

*  **Corelight Sensor exporters**: The [Corelight Sensor exporter](https://docs.corelight.com/docs/sensor/sensor/export/goosecops_exp.html) collects log data from the Sensor, and forwards it to Google Security Operations.

File: README.md [hunk 2]
```diff
@@ -4,27 +4,110 @@ This document describes how you can collect Corelight Sensor logs by configuring
...
 
-![Deployment architecture](images/corelight_parser_arch.png)
+## Ingesting Logs into Google SecOps using Corelight Exporters 
+
+![Deployment architecture](images/parser-corelight-architecture-without-forwarder.png)
 
 The architecture diagram shows the following components:
 
-* **Corelight Sensor**: The system running the [Corelight Sensor](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
+*  **Corelight Sensor**: The system running the [Corelight Sensor ](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
```
Comment (jlagermann) [inline README.md:27]: Please change this line to:

*  **Corelight Sensor**: The system running the [Corelight Sensor ](https://docs.corelight.com/docs/sensor/sensor/quickstarts/quickstart.html).

File: README.md [hunk 3]
```diff
@@ -4,27 +4,110 @@ This document describes how you can collect Corelight Sensor logs by configuring
...
+
+  ![Deployment architecture](images/dynamic-exporter-step-3.png)
+
+## Ingesting Logs into Google SecOps Using a Forwarder 
+
+  ![Deployment architecture](images/parser-corelight-architecture.png)
+
+The architecture diagram shows the following components:
+
+*  **Corelight Sensor**: The system running the [Corelight Sensor ](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
```
Comment (jlagermann) [inline README.md:65]: The link should point to 

https://docs.corelight.com/docs/sensor/sensor/quickstarts/quickstart.html

File: README.md (hunk 1 above)
Comment (poriya-crestdata) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (poriya-crestdata) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (poriya-crestdata) [reply README.md]: Updated.

**Pull request merged**